docs/codeql/codeql-cli/publishing-and-using-codeql-packs.rst

```diff
@@ -72,3 +72,48 @@ The ``analyze`` command will run the default suite of any specified CodeQL packs
 ::
 
    codeql <database> analyze <scope>/<pack> <scope>/<other-pack>
+
+Working with CodeQL packs on GitHub Enterprise Server
+-----------------------------------------------------
+
+.. pull-quote::
+
+   Note
+
+   The Container registry for GitHub Enterprise Server supports CodeQL query packs from GitHub Enterprise Server 3.6 onward.
```

Comment:
Docs for managing the codeql package registry in GHES 3.6 will _not_ be included on github.com. Does this change anything on this line?

Reply:
I don't think it does, but I just want to make sure we are aware that the docs will not be on 3.6.

Reply:
I don't think so — what do you think @felicitymay?

Reply:
I think this is fine. Users reading the codeql docs here will be able to use this feature. Users only reading the docs on github.com are probably not experienced enough with CodeQL to want to uyse this feature anyway.